Gemfile.lock

```diff
@@ -417,6 +417,3 @@ DEPENDENCIES
   uglifier (~> 2.5.3)
   unicode (= 0.4.4)
   unicorn (~> 4.8.3)
-
-BUNDLED WITH
```

Comment:
Are you removing this because of the issues noted here? http://bundler.io/blog/2015/06/24/version-1-10-released.html